ledger_secure_sdk_sys/build.rs

```diff
@@ -209,26 +209,31 @@ fn retrieve_target_file_infos(
 
 /// Fetch the appropriate C SDK to build
 fn clone_sdk(device: &Device) -> PathBuf {
-    let (repo_url, sdk_branch) = match device {
+    let (repo_url, sdk_branch, commit_hash) = match device {
         Device::NanoS => (
             Path::new("https://github.com/LedgerHQ/ledger-secure-sdk"),
             "API_LEVEL_LNS",
+            "93fa69b170dd819c6b93d5cf49dd21acee51ffd0",
```

Comment:
We want to use the head of the branch when cloning the C-SDK (this is what is done when deploying an app for instance).